www/waterfall_view/src/styles/styles.less

```diff
@@ -219,3 +219,7 @@
     }
   }
 }
+
+.svg-tooltip {
+  pointer-events: 'none';
```

Comment:
Any reason why this is not just:

    pointer-events: none;

https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events

Reply:
@Lekensteyn I changed the syntax to match the format on the developer.mozilla.org web site